charts/datasaur/values.yaml

```diff
@@ -160,16 +162,159 @@ jobs:
 
 loki:
   enabled: true
+  alerting:
+    contactPoints.yaml: |
+      apiVersion: 1
+
+      contactPoints:
+        - orgId: 1
+          name: cp_1
+          receivers:
+            - uid: webhook_receiver
+              type: webhook
+              settings:
+                url: https://webhook.site/8bb1d771-b0b3-4867-b680-b67471220f43
+                httpMethod: POST
+            - uid: slack_receiver
+              type: slack
+              settings:
+                url: https://hooks.slack.com/services/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
+        - orgId: 1
+          name: cp_2
+          receivers:
+            - uid: webhook_receiver_2
+              type: webhook
+              settings:
+                url: https://webhook.site/8bb1d771-b0b3-4867-b680-b67471220f43
+                httpMethod: POST
+    alertRules.yaml: |
+      apiVersion: 1
+      groups:
+        - orgId: 1
+          name: General
+          folder: Default
+          interval: 10s
+          rules:
+            - uid: d8395a7e-8498-4e85-ab91-d3144f37d90c
+              title: Backend Error Alert
+              condition: C
+              data:
+                - refId: A
+                  queryType: range
+                  relativeTimeRange:
+                    from: 600
+                    to: 0
+                  datasourceUid: P8E80F9AEF21F6940
```

Comment:
where do we get this `datasourceUid` from?

Reply:
nah kalo ini, i exported from Console di Grafana @tonoman3g I think we can make documentation on how to do it

Reply:
sure